src/tests/ftest/util/general_utils.py

```diff
@@ -40,6 +40,75 @@ class DaosTestError(Exception):
     """DAOS API exception class."""
 
 
+def run_command(command, timeout=60, verbose=True, raise_exception=True,
+                output_check="combined", env=None):
+    """Run the command on the local host.
+
+    This method uses the avocado.utils.process.run() method to run the specified
+    command string on the local host using subprocess.Popen(). Even though the
+    command is specified as a string, since shell=False is passed to process.run
+    it will use shlex.spit() to break up the command into a list before it is
+    passed to subprocess.Popen. The shell=False is forced for security.
+
+    As a result typically any command containing ";", "|", "&&", etc. will fail.
+    This can be avoided in command strings like "for x in a b; echo $x; done"
+    by using "/usr/bin/bash -c 'for x in a b; echo $x; done'".
```

Comment:
Removing the comment.

Reply:
If the ability to use a shell here is not being detected by Bandit, and we are accepting Bandit's assertion that using shells with unvalidated arguments as a valid problem, I wonder if this function should be trying to detect people doing an end-run by passing in shell commands to this function.

Does Bandit detect this kind of end-run using `subprocess`?  I.e. if you encode the use of the shell in the command, rather than just using the command and `Shell=true` does Bandit detect that?  If so, seems like Bandit ought to be extensible in the ability to tell it of other functions/methods and their arguments to which it should apply this check.